src/routes/safe/components/Transactions/TxList/MethodValue.tsx

```diff
@@ -55,6 +56,10 @@ const Value = ({ type, ...props }: RenderValueProps): React.ReactElement => {
         [
         <NestedWrapper>
           {(props.value as string[]).map((address, index) => {
+            if (Array.isArray(address)) {
+              const newProps = { type, ...props, value: address, key: `${props.method}-value-${index}` }
```

Comment:
Just looking back at this. The `key` is not used in the child `map()` in either `newProps.key` or `<PrefixedEthHashInfo key={...}`. If there was a 3D array, the `key` warning would show in the console.